src/main/java/org/broadinstitute/hellbender/tools/exome/orientationbiasvariantfilter/OrientationBiasUtils.java

```diff
@@ -107,19 +105,14 @@ public static boolean isGenotypeInTransitionWithComplement(final Genotype g, fin
         Utils.nonNull(g, "Genotype cannot be null");
         Utils.nonNull(transition, "Transition cannot be null");
 
-        if (g.getAlleles().size() != 2) {
-            throw new UserException.BadInput("Cannot run this method on non-diploid genotypes.");
-        }
-        final boolean isInTransition = g.getAllele(0).getDisplayString().equals(String.valueOf(transition.ref())) &&
-                g.getAllele(1).getDisplayString().equals(String.valueOf(transition.call()));
+        final boolean isInTransition = isGenotypeInTransition(g, transition);
```

Comment:
This would look more readable to me as `return isGenotypeInTransition(g, transition) || is GenotypeInTransition(g, transition.complement())`.